### ISSUE TITLE
Tests do not pass

### ISSUE THREAD

**Issue by jagarsoft** [jagarsoft/mysql2sqlite#1]
This is output of shtest. Please note the following points:

1. I cloned a fresh copy from @frrad's repository
1. bin\shtest.sh in current project is temporary only
1. I didn't re-run unit_tests
1. I suited tests according files splitted into pieces by frrad: *.sql, *.out (*.out will be expected.out)
1. I re-run mysql2sqlite from those *.sql > actual.out
1. I compared expected.out vs actual.out and they should be the same, but they aren't

Tests were run on Windows 10 git-bash and WSL (Windows 10 Subsystem Linux) respectively

Anything is wrong in my environment?

Some ideas???

![output_from_git-bash](https://user-images.githubusercontent.com/3843371/63218046-ecbf9100-c151-11e9-8406-461b2ce26410.png)

![output_from_ubuntu-win10-subsystem](https://user-images.githubusercontent.com/3843371/63218051-ff39ca80-c151-11e9-95fe-61a2a316f7f2.png)

Thanks in advance!!

**Comment by frrad** [comment]
First of all, the salient question is probably "what implementation of `awk` are you running" in both cases. Ironically I think the specific command to get version may depend on version, but try `awk --version` or `awk -W version`.

I investigated your first screenshot and it appears to be caused by the fact that I used `mawk` to generate the expected output files. Apparently `mawk` doesn't support repeat groups and so the expected output which I committed was incorrect. To fix, you should just be able to merge in the `master` branch from my repo. I fixed my fork in https://github.com/frrad/mysql2sqlite/pull/3/files. I have also created a pull request https://github.com/dumblob/mysql2sqlite/pull/61 to try to upstream this fix.

I was unable to repro the second screenshot, but knowing which `awk` created it would help.

**Comment by jagarsoft** [comment]
Thanks for your kind words ;-)

TL;DR;
gawk is always used.
mawk in mysql2sqlite does no difference.
Under git-bash, tests pass.
Under WSL, tests does not pass yet.

First screenshot comes from git-bash:

```shell
git --version
git version 2.22.0.windows.1

ll "`where awk`"
-rwxr-xr-x 1 root 197121 627466 jun.  8 05:43 'C:\Program Files\Git\usr\bin\awk.exe'*

 ll "C:\Program Files\Git\usr\bin" | grep awk
-rwxr-xr-x 1 root 197121  627466 jun.  8 05:43 awk.exe*
-rwxr-xr-x 1 root 197121  627466 jun.  8 05:43 gawk.exe*
-rwxr-xr-x 1 root 197121  627466 jun.  8 05:43 gawk-5.0.0.exe*

awk --version
GNU Awk 5.0.0, API: 2.0 (GNU MPFR 4.0.2, GNU MP 6.1.2)
```

Re-run tests, tests pass:

![output_from_git-bash-2](https://user-images.githubusercontent.com/3843371/63390088-17a62100-c3ae-11e9-85e1-8ec11eab6494.png)

Except for warning, which does not occur in WSL (why?). If you could give me some light, it would be appreciated

Second screenshot comes from WSL (Windows 10 Subsystem Linux):

```shell
uname -a
Linux MyHost 4.4.0-17134-Microsoft #706-Microsoft Mon Apr 01 18:13:00 PST 2019 x86_64 x86_64 x86_64 GNU/Linux

ll $(whereis `apropos awk | cut -d " " -f 1`)
lrwxrwxrwx 1 root root     21 mar 30  2017 /usr/bin/awk -> /etc/alternatives/awk*
-rwxr-xr-x 1 root root 538224 jul  2  2013 /usr/bin/dgawk*
-rwxr-xr-x 1 root root 441512 jul  2  2013 /usr/bin/gawk*
-rwxr-xr-x 1 root root   3188 jul  2  2013 /usr/bin/igawk*
-rwxr-xr-x 1 root root 117768 mar 24  2014 /usr/bin/mawk*
lrwxrwxrwx 1 root root     22 mar 30  2017 /usr/bin/nawk -> /etc/alternatives/nawk*
-rwxr-xr-x 1 root root 445608 jul  2  2013 /usr/bin/pgawk*
lrwxrwxrwx 1 root root     21 mar 30  2017 /usr/bin/X11/awk -> /etc/alternatives/awk*
-rwxr-xr-x 1 root root 538224 jul  2  2013 /usr/bin/X11/dgawk*
-rwxr-xr-x 1 root root 441512 jul  2  2013 /usr/bin/X11/gawk*
-rwxr-xr-x 1 root root   3188 jul  2  2013 /usr/bin/X11/igawk*
-rwxr-xr-x 1 root root 117768 mar 24  2014 /usr/bin/X11/mawk*
lrwxrwxrwx 1 root root     22 mar 30  2017 /usr/bin/X11/nawk -> /etc/alternatives/nawk*
-rwxr-xr-x 1 root root 445608 jul  2  2013 /usr/bin/X11/pgawk*

ll /etc/alternatives/*awk*
lrwxrwxrwx 1 root root   13 mar 30  2017 awk -> /usr/bin/gawk*
lrwxrwxrwx 1 root root   13 mar 30  2017 nawk -> /usr/bin/gawk*

awk --version
GNU Awk 4.0.1
```

Unfortunately, tests does not pass yet
Output was exactly the same.

@dumble: As far as [shell-test](https://github.com/jagarsoft/shell-test) is concerned, although is far to be finished, I hope it fulfils the very first needs. I'd love to know your opinion ;-)

**Comment by frrad** [comment]
I think the warnings are caused by unnecessary escaping. See https://github.com/dumblob/mysql2sqlite/pull/62 (your comment there would be appreciated) and try merging [this commit from ](https://github.com/frrad/mysql2sqlite/commit/f25d8d1d678ff05efb890aa7740f57bafc616648) from my fork into your branch. 

The fact that gawk 4.0.1 has such weird errors is still super mysterious to me. Sorry!

btw I think you meant to @dumblob instead of @.dumble above

**Comment by jagarsoft** [comment]
Thanks @frrad 
Great work!

![image](https://user-images.githubusercontent.com/3843371/63538303-e21d4700-c517-11e9-90a4-f99155052080.png)

PS: Yes, I meant dumblob !!! 😄
